04-cs-fundamentals/classroom/maths-exercises/essential-maths-exercises.md

```diff
@@ -0,0 +1,282 @@
+# Essential Mathematics - practice problems
+_CS fundamentals 0_
+
+## Overview
+Complete each problem manually (i.e. using paper and pencil) and tally your solution with the one provided for each problem below.
+
+### Sorted items
+For each of the following list of objects, share whether the objects are: _sorted in ascending order_, _sorted in descending order_, _unsorted_.
+
+1. 
+[**2, 4, 6, 8, 10**]
+<details><summary>click here to check your answer</summary>
+These numbers are _sorted in ascending order_.
```

Comment:
When I do a preview of this document on Github, it seems like the italics don't carry through, and I still see the underscores. Is this the case for you?

Reply:
Good catch! More nuances of the details tag. Let me fix those.